### DISCUSSION TITLE
Problems with the installation of "crazyradio" under Win11 with VM and RaspberryPy5

### DISCUSSION THREAD

**Discussion by Xdreid** [bitcraze/discussions#1701] [General]
Win11 problem:
The use of 'zadig-2.9.exe' was unsuccessful in installing the drivers. 'Crazyradio' is displayed in the VM but not recognized.

RaspberryPy5:
I am following the instructions at: 
https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/

I receive the following errors from 'Bookworm' after installing 'pipx install --system-site-package':
cannot determin package name form spec "/home/pi". Check package spec for errors.

Can someone help me further?

**Comment by gemenerik** [comment]
We don't fully support rpi, but I'll try to answer. You should run

'pipx install --system-site-packages .'

So with the s and .

Run this in the cloned repo folder
